exercises/anagram/anagram_test.rb

```diff
@@ -7,95 +7,111 @@
 # 3dfde31
 class AnagramTest < Minitest::Test
   def test_no_matches
-     # skip
-     assert_equal [], Anagram.new("diaper").match(["hello", "world", "zombies", "pants"])
+    # skip
+    
+    assert_equal [].sort, Anagram.new('diaper').match(["hello", "world", "zombies", "pants"]).sort
   end
 
   def test_detects_simple_anagram
-     skip
-     assert_equal ["tan"], Anagram.new("ant").match(["tan", "stand", "at"])
+    skip
+    
+    assert_equal ["tan"].sort, Anagram.new('ant').match(["tan", "stand", "at"]).sort
   end
 
   def test_does_not_detect_false_positives
-     skip
-     assert_equal [], Anagram.new("galea").match(["eagle"])
+    skip
+    
+    assert_equal [].sort, Anagram.new('galea').match(["eagle"]).sort
   end
 
   def test_detects_multiple_anagrams
-     skip
-     assert_equal ["stream", "maters"], Anagram.new("master").match(["stream", "pigeon", "maters"])
+    skip
+    
+    assert_equal ["stream", "maters"].sort, Anagram.new('master').match(["stream", "pigeon", "maters"]).sort
   end
 
   def test_does_not_detect_anagram_subsets
-     skip
-     assert_equal [], Anagram.new("good").match(["dog", "goody"])
+    skip
+    
+    assert_equal [].sort, Anagram.new('good').match(["dog", "goody"]).sort
   end
 
   def test_detects_anagram
-     skip
-     assert_equal ["inlets"], Anagram.new("listen").match(["enlists", "google", "inlets", "banana"])
+    skip
+    
+    assert_equal ["inlets"].sort, Anagram.new('listen').match(["enlists", "google", "inlets", "banana"]).sort
   end
 
   def test_detects_multiple_anagrams
-     skip
-     assert_equal ["gallery", "regally", "largely"], Anagram.new("allergy").match(["gallery", "ballerina", "regally", "clergy", "largely", "leading"])
+    skip
+    
+    assert_equal ["gallery", "regally", "largely"].sort, Anagram.new('allergy').match(["gallery", "ballerina", "regally", "clergy", "largely", "leading"]).sort
```

Comment:
You didn't address my other question:
> Why have you changed from the old 3 line "detector, anagrams, assertion" format?

That would help avoid situations like this where you end up with one really long line.